submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,302 @@
+html, body {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    padding: 20px 0;
+    position: relative;
+}
+
+.top_menu {
+    display: flex;
+    padding: 0 15px;
+    align-items: center;
+}
+
+.top_menu__item {
+    line-height: 30px;
+    padding: 0 10px 0 10px;
+    font-size: 14px;
+    text-decoration: none;
+    color: rgba(0, 0, 0, 0.75);
+}
+
+.top_menu__item-container:hover {
+    text-decoration: underline;
+}
+
+.top_menu__item_bell {
+    margin-left: 5px;
+    width: 20px;
+    height: 25px;
+    display: block;
+    cursor: pointer;
+    background: url("../icons/bell.png") center center no-repeat;
+    background-size: 20px 25px;
+}
+
+.top_menu__item_user {
+    display: block;
+    height: 32px;
+    width: 32px;
+    border-radius: 50%;
+    background: url("../icons/favicon.png") center center no-repeat;
+    background-size: 32px 32px;
+}
+
+.top_menu__item_bell:hover, .top_menu__item_user:hover, .button_popup:hover {
+    opacity: 75%;
```

Comment:
no, bad styling idea, better remove it :) So, you are doing items worse visible to the user

Reply:
Done!